client/example/package.json

```diff
@@ -11,7 +11,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@davidia/component": "workspace:^",
+    "@diamondlightsource/davidia": "workspace:^",
```

Comment:
There should be a corresponding fix in the example application to its imports

Reply:
Right, it's in a separate module of a monorepo so should not use that relative path